src/locale/messages.fr.xlf

```diff
@@ -934,6 +934,12 @@
           <context context-type="sourcefile">src/app/modules/group/pages/managed-groups/managed-groups.component.html</context>
           <context context-type="linenumber">3</context>
         </context-group>
+      </trans-unit><trans-unit id="8471940861423120175" datatype="html">
+        <source>Group successfully added as child group</source><target state="new">Group successfully added as child group</target>
```

Comment:
Please do all the translations you introduced.

Reply:
I merged master into this branch and completed the missing translations